cardy/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "next-cardigan",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "fractal start"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@frctl/fractal": "^1.1.7",
+    "babel-env": "^2.4.1",
+    "babel-plugin-react-require": "^3.0.0",
+    "babel-plugin-transform-react-jsx": "^6.24.1",
+    "babel-preset-env": "^1.6.1",
+    "fractal-react-adapter": "https://github.com/davidpmccormick/fractal-react-adapter.git"
```

Comment:
Are we missing some of the babel plugins we're using [here](https://github.com/wellcometrust/wellcomecollection.org/pull/2046/files#diff-11944165b9979e2476546e4aac30b24bR7)?